tracking/speed_groups.cpp

```diff
@@ -0,0 +1,22 @@
+#include "tracking/speed_groups.hpp"
+
+namespace tracking
+{
+uint32_t const kSpeedGroupThresholdPercentage[] = {8, 16, 33, 58, 83, 100, 100, 100};
```

Comment:
Feel free to add a static assert here, that ARRAY_SIZE of kSpeedGroup...ge is SpeedGroup::Count.